.github/workflows/continuous-integration.yml

```diff
@@ -33,8 +33,8 @@ jobs:
       - name: Install missed dependencies
         run: poetry install
 
-      - name: Format with black
-        run: poetry run black . --check --diff
+      #- name: Format with black
+      #  run: poetry run black . --check --diff
```

Comment:
@MarlonMueller Why do you need to format it in the CI? The CI doesn't add new commits.

Reply:
It just enforces that the files are formatted in the first place.

Reply:
Aah! Thank you, that's a good idea 🙌